src/wp-content/themes/twentyfourteen/functions.php

```diff
@@ -325,7 +331,8 @@ function twentyfourteen_font_url() {
  */
 function twentyfourteen_scripts() {
 	// Add Lato font, used in the main stylesheet.
-	wp_enqueue_style( 'twentyfourteen-lato', twentyfourteen_font_url(), array(), null );
+	$font_version = ( 0 === strpos( (string) twentyfourteen_font_url(), get_template_directory_uri() . '/' ) ) ? '20230328' : null;
```

Comment:
Can we use `str_starts_with` here?

Reply:
Unfortunately, no. These themes should work with older versions of both PHP and WordPress.